### ISSUE TITLE
video.js was not found!

### ISSUE THREAD

**Issue by 82253452** [LonnyGomes/vjs-video#31]
i use the webpack  for 

require('../../node_modules/video.js/dist/video.min.js');
require('../../node_modules/video.js/dist/video-js.min.css');
require('../../node_modules/vjs-video/dist/vjs-video.min.js');

but has error 
Error: video.js was not found!

i don't know wat't wrong~~

**Comment by ghost** [comment]
@NateRudolph has a bit more experience with webpack than I. My hunch is that webpack isn't aware of `vis-video's` dependency on `video.js`. With that said, I can work to make `vis-video` more module friendly in the future.

**Comment by neiltron** [comment]
It looks like the error is happening because vjs-video expects videojs to be in the global scope. See: https://github.com/LonnyGomes/vjs-video/blob/d1e2ac5b3e3ddb21d4d99b178cac25fb64c39098/app/scripts/directives/vjs.directive.js#L30-L31

You should be able to fix this by just doing `window.videojs = videojs;` immediately after your `require`/`import` statement.

**Comment by ghost** [comment]
@neiltron good call! I will work to address that in #42.

**Comment by ghost** [comment]
[This](https://github.com/videojs/video.js/issues/2750) video.js thread is also interesting in terms of using webpack.

**Comment by ghost** [comment]
FYI, this issue was resolve in v0.1.7 of `vjs-video` (see #42). You shouldn't have to use @neiltron's workaround any more.

**Comment by aywengo** [comment]
The issue is still present in 0.1.9

```
ERROR in ../~/vjs-video/dist/vjs-video.min.js
Module not found: Error: Cannot resolve module 'videojs' in 
 @ ../~/vjs-video/dist/vjs-video.min.js 1:72-103

```

**Comment by ghost** [comment]
@aywengo it turns out this is a regression from trying to fix a bug with AMD module loading. I'll handle the fix in #64.

**Comment by ghost** [comment]
@aywengo Fixed in v0.1.10
